lms/djangoapps/courseware/views/views.py

```diff
@@ -736,8 +749,18 @@ def course_about(request, course_id):
         if configuration_helpers.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False)):
             return redirect(reverse(course_home_url_name(course.id), args=[unicode(course.id)]))
 
+        if 'cpa_cookie' in request.COOKIES:
+            cpa_cookie = request.COOKIES.get('cpa_cookie')
+            try:
+                cpa_cookie = literal_eval(cpa_cookie)
+            except ValueError:
+                pass
```

Comment:
Is this error handling really useful? If yes, maybe we should at least log the error?

Reply:
Yes, you're right, it's better to log some useful information to debug easily, but it's not critical there.